README.md

```diff
@@ -6,36 +6,67 @@ A tool for validating community plugins for publishing to Grafana.com.
 
 The tool expects path to either a remote or a local ZIP archive.
 
-## Install
+## Install and usage
+
+### With docker (recommended\*)
 
 ```SHELL
-cd pkg/cmd/plugincheck2
-go install
+docker run --pull=always grafana/plugin-validator-cli -sourceCodeUri [source_code_location/] [plugin_archive.zip]
+```
+
+Example:
+
+```SHELL
+docker run --pull=always grafana/plugin-validator-cli -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
 ```
 
-## Run
+> \* Docker is recommended because the image contains all the security scanning tools for the validator such as semgrep, gosec and osv-scanner
```

Comment:
instead of listing here you could link to the security tools section which then includes direct links